tools/cockpit.spec

```diff
@@ -100,12 +100,14 @@ BuildRequires: pam-devel
 
 BuildRequires: autoconf automake
 BuildRequires: make
-BuildRequires: /usr/bin/python3
 %if 0%{?rhel} && 0%{?rhel} <= 8
+# RHEL 8 has multiple Python versions, ensure the default one
+BuildRequires: python3-devel
 # RHEL 8's gettext does not yet have metainfo.its
 BuildRequires: gettext >= 0.19.7
 BuildRequires: libappstream-glib-devel
 %else
+BuildRequires: /usr/bin/python3
```

Comment:
If this is left here for Fedora, note, that you should BR python3-devel for Fedora/EPEL as well. From guidelines:

> Every package that uses Python (at runtime and/or build time) and/or installs Python modules MUST explicitly include BuildRequires: python3-devel in its .spec file, even if Python is not actually invoked during build time.

See: https://docs.fedoraproject.org/en-US/packaging-guidelines/Python/#_buildrequire_python3_devel

Reply:
*nod* -- this takes some work to rebuild all our images to include python3-devel. I can also block this until all of them got done, I suppose there's not that much urgency on this PR.

Reply:
All images refreshed.